README.md

```diff
@@ -123,13 +149,74 @@ class E2eTest extends PantherTestCase
         // enjoy the same API for the 3 felines
         // $*client->request('GET', '...')
 
-        $kernel = static::createKernel(); // You have also access to the app's kernel
+        $kernel = static::createKernel(); // If you are testing a Symfony app, you also have access to the kernel
 
         // ...
     }
 }
 ```
 
+### Creating Isolated Browsers to Test Apps Using [Mercure](https://mercure.rocks) or WebSockets
```

Comment:
```suggestion
### Creating Isolated Browsers to Test Apps Using [Mercure](https://mercure.rocks) or WebSocket
```

See: https://tools.ietf.org/html/rfc6455

Reply:
MDN uses Websockets, but ok

Reply:
It actually says:

>  The WebSocket API (WebSockets)

Haha...